src/table/merge_iterator.rs

```diff
@@ -281,47 +289,95 @@ mod tests {
             .collect()
     }
 
-    fn check_sequence(mut iter: Box<Iterators>, n: usize) {
+    fn check_sequence_both(mut iter: Box<Iterators>, n: usize, reversed: bool) {
+        // test sequentially iterate
         let mut cnt = 0;
         iter.rewind();
         while iter.valid() {
-            assert_eq!(user_key(iter.key()), format!("{:012x}", cnt).as_bytes());
+            let check_cnt = if reversed { n - 1 - cnt } else { cnt };
+            assert_eq!(
+                Bytes::copy_from_slice(user_key(iter.key())),
```

Comment:
Why clone?

Reply:
When asserting eq, `Bytes` is more readable than `&[u8]` in output, as it will show characters as-is and only escape control characters. This performance penalty is acceptable as it's only used in tests.

Reply:
Cool, how about making it an explicit function and comment the intentions?

Reply:
I have added `assert_bytes_eq`, which first cast `&[u8]` into `Bytes`, and then `assert_eq!`.